aka.js

```diff
@@ -324,6 +350,155 @@ function spawn_update_check(update_file_path) {
   }).unref();
 }
 
+// Get random tips or update available statement (if applicable)
+function get_epilogue() {
+  const update_available = module.exports.update_available ? (Object.keys(module.exports.update_available).length !== 0) : false;
+  if (update_available) {
+    return module.exports.terminal.update_statement(module.exports.update_available.current, module.exports.update_available.latest);
+  } else {
+    return module.exports.random_tips[Math.floor(module.exports.random_tips.length * Math.random())]
+  }
+}
+
+// Print UI and any error messages
+function print_ui(cliui, errorMessage) {
+  if (errorMessage) {
+    cliui.span(`\n${errorMessage}`);
+  } else {
+    cliui.span(`\n${get_epilogue()}`);
+  }
+  console.log(cliui.toString());
+}
+
+// Print help for a specific command group
+function print_group_help(argv, group) {
+  // Initialize UI
+  let errorMessage = ''
+  const ui = require('cliui')({width: process.stdout.columns})
+  ui.div(chalk.bold('\nAkkeris CLI Help\n'))
+
+  // Reset yargs so we can initialize it with only the plugin that we want
+  const newYargs = require('yargs');
+
+  if (!init_plugins(module, module.exports.config.plugins_dir, group)) {
+    if (!init_plugins(module, module.exports.config.third_party_plugins_dir, group)) {
+      console.log(module.exports.terminal.markdown(`\n !!▸!! Bad bad error. You should never see this\n`));
+      return;
+    }
+  }
+
+  // A specific command was provided along with the group name
+  if (argv.group.length > 1) {
+    // Verify that the specific command is valid
+    const givenCommand = argv.group[1];
+    const foundCommand = newYargs.getUsageInstance().getCommands().filter(a => a[0].split(" ").find(b => b === givenCommand))
+
+    // Tell Yargs to run the specific command with the '--help' flag 
+    if (foundCommand && foundCommand.length > 0) {
+      newYargs.help(true).parse(`${foundCommand[0][0]} --help`)
+    } else {
+      errorMessage = `${chalk.red.italic("Invalid command:")} ${givenCommand}`
+    }
+  }
+
+  // Render the name of the group
+  ui.div(chalk.italic(capitalize(group)))
+
+  // Render all of the group commands
+  const commands = newYargs.getUsageInstance().getCommands().sort((a, b) => a[0] < b[0] ? -1 : 1);
+  const width = commands.reduce((acc, curr) => Math.max(stringWidth(`${argv["$0"]} ${curr[0]}`), acc), 0) + 6;
```

Comment:
Do we need stringWidth? Can we just use `.length` rather than `stringWidth`. We're trying to cap or limit our dependencies unless there's a strong reason to use it, `.length` is probably good enough for 99.999% of the use cases, unless I'm missing something.

Reply:
Discussed in-person, keeping string-width but removing chalk.